### PR TITLE
refactor: use renderable guards for tab content

### PR DESCRIPTION
**Pull request by QDyanbing**
## 说明

- 使用 `isReactRenderable` 统一判断 tab label、icon、extra content 以及增删图标
- 统一 fallback 行为并支持数字 `0` 等有效 React 内容
- 将 `@rc-component/util` 的最低版本提升到 `^1.13.0`

## 验证

- `npm run tsc`
- `npm run lint`
- `npm test -- tests/index.test.tsx --runInBand`

## Summary by CodeRabbit

- **Bug 修复**
  - 修复标签页图标、标签文本、额外内容及操作按钮图标的渲染判断。
  - 现在可正确显示数字等有效的 React 内容，包括值为 `0` 的图标或内容。
  - 当新增或移除图标为不可渲染值（如空字符串、`false` 或 `null`）时，分别回退显示默认的“+”或“×”图标。
  - 下拉菜单中的移除按钮也能正确显示数字图标。

### CODE REVIEW

**Review by yoyo837 — CHANGES_REQUESTED** [review]
The implementation changes look semantically reasonable, but the coverage checks are currently failing (`codecov/patch` and `codecov/project`). The new icon renderability/fallback paths in `OperationNode` and `TabNode`, including numeric zero, are not fully covered.

Please add focused tests for the new branches—especially zero-valued icons and the false/empty/nullish fallback behavior—so the coverage gate passes before this is merged.

**Comment by QDyanbing** [comment]
Addressed in fa9866a. Focused tests now cover the new `OperationNode` and `TabNode` close/remove icon branches, including numeric zero and false, empty-string, and nullish fallbacks. The full test suite (80 tests) passes, and all renderability branches changed by this PR are hit locally.

File: tests/overflow.test.tsx [hunk 1]
```diff
@@ -398,6 +398,51 @@ describe('Tabs.Overflow', () => {
...
+        jest.runAllTimers();
+      });
+
+      const removes = document.querySelectorAll('.rc-tabs-dropdown-menu-item-remove');
+      expect(removes).toHaveLength(2);
+      expect(removes[0]).toHaveTextContent('0');
+      expect(removes[1]).toHaveTextContent('0');
+
+      unmount();
+      jest.useRealTimers();
```
Comment (yoyo837) [inline tests/overflow.test.tsx]: Non-blocking nit: if an assertion fails mid-test, the trailing `jest.useRealTimers()` never runs and fake timers leak into the tests that follow. Wrapping the body in `try/finally` keeps the cleanup on failure paths too:

```suggestion
      jest.useFakeTimers();
      try {
        hackOffsetInfo.container = 20;

        const onEdit = jest.fn();
        const { container, unmount } = render(
          getTabs({
            editable: { onEdit, removeIcon: 0 },
            items: [
              {
                key: 'close-icon',
                label: 'Close icon',
                closeIcon: 0,
                children: 'Close icon',
              },
              {
                key: 'remove-icon',
                label: 'Remove icon',
                closeIcon: '',
                closable: true,
                children: 'Remove icon',
              },
            ],
          }),
        );

        triggerResize(container);
        act(() => {
          jest.runAllTimers();
        });

        fireEvent.mouseEnter(container.querySelector('.rc-tabs-nav-more'));
        act(() => {
          jest.runAllTimers();
        });

        const removes = document.querySelectorAll('.rc-tabs-dropdown-menu-item-remove');
        expect(removes).toHaveLength(2);
        expect(removes[0]).toHaveTextContent('0');
        expect(removes[1]).toHaveTextContent('0');

        unmount();
      } finally {
        jest.useRealTimers();
      }
```

**Review by yoyo837 — COMMENTED** [review]
Thanks for the quick turnaround - the coverage concerns are resolved. One non-blocking test-hygiene suggestion below; happy to merge once it is applied.

File: tests/overflow.test.tsx (hunk 1 above)
Comment (QDyanbing) [reply tests/overflow.test.tsx]: Addressed in bb851a5. The test body is now wrapped in `try/finally`, so `jest.useRealTimers()` always runs even when an assertion or interaction fails. I also reran the full test suite and TypeScript checks successfully.

File: tests/overflow.test.tsx [hunk 2]
```diff
@@ -398,6 +398,51 @@ describe('Tabs.Overflow', () => {
       });
     });
 
+    it('renders numeric remove icons in the dropdown menu', () => {
+      jest.useFakeTimers();
```
[the remark this answers is not in the document]
Comment (QDyanbing) [reply tests/overflow.test.tsx:402]: Addressed in bb851a5 by moving `jest.useRealTimers()` into a `finally` block. Full tests, TypeScript, and formatting checks pass.
